### PR TITLE
Restructure folders and procjects.

### PR DESCRIPTION
**Pull request by AHdevelopment**
I have restructed the solution and ran the tests (100% pass). I have also added an unit test project for the Process Manager. I will talk to Lasse about the number of test projects, before moving the actual tests. 

image

image

image

### CODE REVIEW

**Review by BjarkeMeier — DISMISSED** [review]

File: source/dotnet/Services/ProcessManager.UnitTests/Test1.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿// Copyright 2020 Energinet DataHub A/S
```
Comment (BjarkeMeier) [inline source/dotnet/Services/ProcessManager.UnitTests/Test1.cs:1]: nit: Perhaps remove this template-generated file?

**Review by BjarkeMeier — COMMENTED** [review]
This structure aligns nicely with [ADR-102](https://energinet.atlassian.net/wiki/spaces/D3/pages/290390042/ADR+102+-+Folderstruktur). We have a separate story to align with ADR-102, but I think it might be relevant for you to know of its existence.

**Review by lasrinnil — COMMENTED** [review]
What about the namespaces? Are they supposed to come in another PR?

**Comment by BjarkeMeier** [comment]
> What about the namespaces? Are they supposed to come in another PR?

What namespaces do you have in mind?

**Review by lasrinnil — APPROVED** [review]

**Pull request merged**